instrumentation/aws-lambda/aws-lambda-core-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awslambdacore/v1_0/AwsLambdaRequestHandlerInstrumentation.java

```diff
@@ -90,7 +93,7 @@ public static void stopSpan(
         functionInstrumenter().end(functionContext, input, null, throwable);
       }
 
-      OpenTelemetrySdkAccess.forceFlush(1, TimeUnit.SECONDS);
+      OpenTelemetrySdkAccess.forceFlush(flushTimeoutNanos, TimeUnit.NANOSECONDS);
```

Comment:
```suggestion
      OpenTelemetrySdkAccess.forceFlush(flushTimeout(), TimeUnit.SECONDS);
```

Reply:
OTEL_LAMBDA_FLUSH_TIMEOUT_DEFAULT is 10 seconds.

Reply:
Yes, OTEL_LAMBDA_FLUSH_TIMEOUT_DEFAULT is 10, that is a braking change from 1 second. 
How to add proper changelog?